tests/models/instructblip/test_modeling_instructblip.py

```diff
@@ -521,6 +521,8 @@ def prepare_img():
 @require_torch
 @slow
 class InstructBlipModelIntegrationTest(unittest.TestCase):
+    # TODO (@Younes): Re-enable this when 8-bit or 4-bit is implemented.
+    @unittest.skip(reason="GPU OOM")
```

Comment:
I tried `device_map="auto"` or `(b)float16`. Always got an error. With `device_map=auto`, the error doesn't say OOM but instead of `RuntimeError: GET was unable to find an engine to execute this computation`.

However, Google tells me it's still kind of GPU memory issue.

Reply:
cc @younesbelkada  for 4 or 8 bit

Reply:
Sure, will add it after https://github.com/huggingface/transformers/pull/24490 gets merged